SU-HSDO/stanford_mrc
# MRCD8-171 visitors list

# READY FOR REVIEW


# Summary
- Visitors list styles and template

# Needed By (Date)
- End of sprint (1/17), if possible

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Verify styles and pattern is working

# Affected Projects or Products
- MRC
- stanford_mrc

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-171

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
